test/suites/rpm/install-and-upgrade-successful.robot

```diff
@@ -101,7 +98,7 @@ Enable MicroShift Dependency Repositories
     ...    subscription enabled.
     ${uname}=    Command Should Work    uname -m
     Command Should Work
-    ...    subscription-manager repos --enable rhocp-${DEPENDENCY_VERSION}-for-rhel-9-${uname}-rpms --enable fast-datapath-for-rhel-9-${uname}-rpms
+    ...    subscription-manager repos --enable fast-datapath-for-rhel-9-${uname}-rpms
```

Comment:
Since we're conditionally enabling rhocp from the hypervisor, should we do the same for fast-datapath?
It should not be necessary for mirror repos.